src/Upgrader/Business/CommandExecutor/CommandResultDto.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * Copyright © 2021-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace Upgrader\Business\CommandExecutor;
+
+class CommandResultDto
```

Comment:
Can we avoid DTO postfix and rename it e.g to CommandOutputResult or smth similar?

Reply:
Renamed to CommandResult